airflow/www/static/js/dag/details/taskInstance/ExtraLinks.tsx

```diff
@@ -57,8 +57,15 @@ const ExtraLinks = ({
     if (!url) {
       return true;
     }
-    const urlRegex = /^(https?:)/i;
-    return urlRegex.test(url);
+    const path = new URL(url, "http://localhost");
```

Comment:
Hmm. I think this won't work for absolute URLs now

```
URL("http://www.apache.org", "http://localhost")
```

-> `URL("http://www.apache.org", "http://localhost")`

Reply:
Ahh.. stupid me